src/components/vtk/VtkSegmentationSliceRepresentation.vue

```diff
@@ -135,6 +138,35 @@ const applySegmentColoring = () => {
 
 watchEffect(applySegmentColoring);
 
+const configStore = useSegmentGroupConfigStore();
+const config = computed(() =>
+  configStore.getConfig(unref(viewId), unref(segmentationId))
+);
+
+const outlineThickness = computed(() => config.value?.outlineThickness ?? 2);
+// @ts-expect-error vtk.js types are incomplete
```

Comment:
Can you please make a PR against vtk.js to fix these types?

Reply:
https://github.com/Kitware/vtk-js/pull/3174